src/Desktop/UIAutomation/DesktopElementHelper.cs

```diff
@@ -1,66 +1,24 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
-using Axe.Windows.Core.Types;
 using System;
 using System.Collections.Generic;
 using UIAutomationClient;
 
 namespace Axe.Windows.Desktop.UIAutomation
 {
     /// <summary>
-    /// Class DesktopElementHelper
-    /// it is a helper class for DesktopElement to handle property/pattern related caching and more
-    /// it is a singleton object via static members
+    /// A helper class to manage the cached properties that we always request from UIA
     /// </summary>
     public static class DesktopElementHelper
     {
-
```

Comment:
These changes mirror those in #550. I'll plan on merging #550 first, then handle the merge into this PR